src/hooks/useSwapDerivedOutputs.ts

```diff
@@ -35,6 +32,7 @@ import {
 import { ethereumUtils } from '@rainbow-me/utils';
 import Logger from '@rainbow-me/utils/logger';
 
+const SWAP_POLLING_INTERVAL = 10000;
```

Comment:
Is there any reason we would want to potentially add this value to remote config? Just spitballing.

Reply:
oh maybe? there's another part of this work that we need to add, maybe we could add it there if we want to do it